core/epubmaker/epubmaker.rb

```diff
@@ -158,91 +270,139 @@ def copyInteriorImg(dir, opf, dest)
 			copied << i
 		end
 	end
-	copied
+	return copied
+rescue => logstring
+	return ''
+ensure
+	Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
+
+## wrapping a Mcmlln::Tools method in a new method for this script; to return a result for json_logfile
+def moveFileToDoneFolder(file, dest, logkey='')
+	Mcmlln::Tools.moveFile(file, dest)
+rescue => logstring
+ensure
+	Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
+
+## wrapping Bkmkr::Tools.runpython in a new method for this script; to return a result for json_logfile
+def zipEpub(script, args, logkey='')
+	Bkmkr::Tools.runpython(script, args)
+rescue => logstring
+ensure
+    Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
+end
+
+## wrapping a Mcmlln::Tools method in a new method for this script; to return a result for json_logfile
+def deleteTmpEpub(file, logkey='')
+	Mcmlln::Tools.deleteFile(file)
+rescue => logstring
+ensure
+    Mcmlln::Tools.logtoJson(@log_hash, logkey, logstring)
 end
 
 # ---------------------- PROCESSES
-deleteOld(OEBPS_dir)
-deleteOld(METAINF_dir)
+
+data_hash = readConfigJson('read_config_json')
+
+##### local definition(s) based on data from config.json
+cover = data_hash['frontcover']
+
+# cover html file within the epub
+cover_html = File.join(oebps_dir, "cover.html")
```

Comment:
I think it was part of some troubleshooting & I forgot to move it back - same with line 322.  I'll move them back! :)